quarkus/service/build.gradle.kts

```diff
@@ -113,6 +108,12 @@ dependencies {
   testImplementation("software.amazon.awssdk:kms")
   testImplementation("software.amazon.awssdk:dynamodb")
 
+  runtimeOnly(project(":polaris-relational-jdbc"))
+  runtimeOnly("io.quarkus:quarkus-jdbc-postgresql") {
```

Comment:
Does `quarlus/service` need this? There is no "runtime" in this module. It can only run as part of the admin tool or the main Quarkus server... did I miss something?

Reply:
Sorry I think i missed this, why do we have server module ? i see eclipse link there this seems to be a natural place to be if thats what we need.

Reply:
I think we do need as the integ refers them removing them fails them, presently there is not integTest for eclipselink it passes, so this would explain why it works

Reply:
tests do need them, but not `runtime`, I guess.

Reply:
`quarkus/service` is supposed to have only the code to implement services' logic. That code binds to core interfaces, so it should not need any specific JDBC dependencies to build.

`quarkus/server` is the final assembly for the server runtime image - this module pulls together all components that may be needed in Polaris runtime.

Reply:
removing this fails to start the service, and hence test fails, so if i just change this from runtime to testImpl it fails.
I can take this as follow-up to find better ways to achieve this.

Reply:
ok, let's deal with that later